inc/class-command.php

```diff
@@ -259,30 +236,124 @@ protected function shell( InputInterface $input, OutputInterface $output ) {
 	}
 
 	/**
-	 * Get the name of the project host names from the local config.
+	 * Command to update the config.local.yaml file and re-provision.
+	 */
+	protected function provision( InputInterface $input, OutputInterface $output ) {
+		$success = $this->write_config_file();
```

Comment:
is `$success` necessary? it's just a temp var

Reply:
I prefer not to call functions from within an if statement if they're carrying out an action at the same time, our engineering guidelines make this recommendation as it's easier to debug and set breakpoints.

Reply:
ah ok TIL - where is that on the engineering handbook?
For some reason I thought we wanted to avoid single use variables

Reply:
Realised it doesn't explicitly mention this so I guess it's a personal preference, apologies.